code/client/munkilib/updatecheck.py

```diff
@@ -3056,8 +3062,11 @@ def check(client_id='', localmanifestpath=None):
         # and active installs/removals
         item_list = list(installinfo.get('optional_installs', []))
         item_list.extend(installinfo['managed_installs'])
-        item_list.extend(installinfo['removals'])
         download_icons(item_list)
+        # We don't need to download iconds for items being removed,
+        # they should alredy be present and if they aren't we're
```

Comment:
But if we're removing Photoshop CS4 and don't already have the icon, why do we want it?